apps/client/src/common/api/db.ts

```diff
@@ -181,6 +181,17 @@ export async function importSpreadsheetPreview(file: File, options: ImportMap):
   return response.data;
 }
 
+export async function getSpreadsheetWorksheetNames(file: File): Promise<string[]> {
```

Comment:
I am unsure I follow this endpoint, we already upload the excel, could we not use that instead?